src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = "CORECLR";
+        private static Architecture? s_processArch = null;
+        private static bool s_is64BitProcess = IntPtr.Size == 8;
+
+        public static string GetRuntimeDebugName()
+        {
+            return RuntimeName;
```

Comment:
So that when people copy and paste exception stacks with data. The version number is captured. It's like seeing a watson report with .NET Framework and no version. The more info, the more useful.